### PR TITLE
[test-runner] Add android RenderTestRunner baseline metrics

### PR DESCRIPTION
**Pull request by zmiao**
All of the metrics are generated from Google Piexl4 that runs on Firebase device farm, since we are running our android test runner with this device on CircleCi with job `next-android-render-test-runner` .
Currently they are saved under `render-test/metrics/android`, the metrics are platform and device dependent.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -27,7 +27,6 @@ xcuserdata
 /platform/ios/benchmark/assets/glyphs/DIN*
 /platform/ios/benchmark/assets/tiles/mapbox.mapbox-terrain-v2,mapbox.mapbox-streets-v6
 /platform/darwin/developer.xcconfig
-**/token
```
Comment (alexshalamov) [inline .gitignore]: Could we add this back? I think it was added for security reasons, so that we don't commit tokens accidentally.

Comment (zmiao) [reply .gitignore]: I see, I will revert this one. Thanks!

File: platform/android/src/test/render_test_runner.cpp [hunk 2]
```diff
@@ -252,26 +252,33 @@ void android_main(struct android_app* app) {
...
-        for (const auto& arg : arguments) {
-            argv.push_back((char*)arg.data());
-        }
-        argv.push_back(nullptr);
+        auto runTestWithManifest = [&](const std::string manifest, const std::string filter = {}) -> bool {
+            std::string configFile = storagePath + manifest;
+            std::vector<std::string> arguments = {"mbgl-render-test-runner", "-p", configFile, filter};
+            std::vector<char*> argv;
+            for (const auto& arg : arguments) {
+                argv.push_back((char*)arg.data());
```
Comment (alexshalamov) [inline platform/android/src/test/render_test_runner.cpp]: nit: static_cast

File: platform/android/src/test/render_test_runner.cpp [hunk 3]
```diff
@@ -252,26 +252,33 @@ void android_main(struct android_app* app) {
...
 
-        std::string configFile = storagePath + "/android-manifest.json";
-        std::vector<std::string> arguments = {"mbgl-render-test-runner", "-p", configFile};
-        std::vector<char*> argv;
-        for (const auto& arg : arguments) {
-            argv.push_back((char*)arg.data());
-        }
-        argv.push_back(nullptr);
+        auto runTestWithManifest = [&](const std::string manifest, const std::string filter = {}) -> bool {
+            std::string configFile = storagePath + manifest;
```
Comment (alexshalamov) [inline platform/android/src/test/render_test_runner.cpp]: nit: const

File: platform/android/src/test/render_test_runner.cpp [hunk 4]
```diff
@@ -252,26 +252,33 @@ void android_main(struct android_app* app) {
...
         unZipFile(env, zipFile, storagePath);
 
-        std::string configFile = storagePath + "/android-manifest.json";
-        std::vector<std::string> arguments = {"mbgl-render-test-runner", "-p", configFile};
-        std::vector<char*> argv;
-        for (const auto& arg : arguments) {
-            argv.push_back((char*)arg.data());
-        }
-        argv.push_back(nullptr);
+        auto runTestWithManifest = [&](const std::string manifest, const std::string filter = {}) -> bool {
```
Comment (alexshalamov) [inline platform/android/src/test/render_test_runner.cpp]: nit: maybe capture only args that are required.

**Pull request merged**